Cargo.lock

```diff
@@ -944,6 +952,12 @@ dependencies = [
  "serde",
 ]
 
+[[package]]
+name = "const_fn"
```

Comment:
is this still a thing? also looks like this is a new transitive dependency introduced by this update

Reply:
(by crossbeam-epoch)